src/mumble/AudioOutputSample.cpp

```diff
@@ -228,7 +228,7 @@ bool AudioOutputSample::prepareSampleBuffer(unsigned int frameCount) {
 	iLastConsume = sampleCount;
 
 	// Check if we can satisfy request with current buffer
-	if (iBufferFilled >= sampleCount)
+	if (iBufferFilled >= sampleCount + INTERAURAL_DELAY)
```

Comment:
It seems that INTERAURAL_DELAY should only be added if positional audio is actually active (not only enabled but actually being used)

This also applies to other parts in the code

Reply:
I don't think there's any harm in doing it if positional audio is inactive? I didn't add a check for this because I thought it might cause performance issues to check if it's enabled all the time.

Reply:
I am not entirely sure what the implications here might be.

@davidebeatrici your thoughts?

Reply:
Any updates on this?

Reply:
@davidebeatrici

Reply:
Adding a check would definitely not cause performance issues, but consider that most of the code should be refactored anyway.

Reply:
I guess if we introduce a check we can run into edge cases for turning this on or off.

My main concern with this is that it might cut away small bits of voice data every time because we are not using all samples.
I don't know if that actually produces any audible artifacts though :thinking:

Reply:
As long as the buffer is clean it shouldn't.

Reply:
I haven't noticed any artifacts when not using positional audio. It seems that the buffer is shifted by the `iLastConsume` variable which is not modified, so I don't think any voice data would be lost.

```cpp
224         // we can not control exactly how many frames decoder returns
225         // so we need a buffer to keep unused frames
226         // shift the buffer, remove decoded and played frames
227         for (unsigned int i = iLastConsume; i < iBufferFilled; ++i)
228                 pfBuffer[i - iLastConsume] = pfBuffer[i];
```

Reply:
Alright then this is fine by me :+1: